src/etc/man/cargo-vendor.1

```diff
@@ -54,13 +54,21 @@ vendored and synced to the output.
 \fB\-\-no\-delete\fP
 .RS 4
 Don\(cqt delete the "vendor" directory when vendoring, but rather keep all
-existing contents of the vendor directory
+existing contents of the vendor directory.
+.RE
+.sp
+\fB\-\-explicit\-version\fP
+.RS 4
+Normally versions are only added to disambiguate multiple versions of the same package.
+This option causes all directories in the "vendor" directory to be explicitly versioned,
+which makes it easier to track the history of vendored packages over time, and can help
+with the performance of re-vendoring when only a subset of the packages have changed.
 .RE
 .sp
 \fB\-\-respect\-source\-config\fP
 .RS 4
 Instead of ignoring \fB[source]\fP configuration by default in \fB.cargo/config\fP
-read it and use it when downloading crates from crates.io, for example
```

Comment:
Please don't make unrelated changes in a PR like this.

Reply:
I split them into their own commit. I can make a separate PR if you want, but it seems a bit much for a 2 char doc change.

Reply:
Having them in their own commit suffices, sure.